src/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.Design/project.json

```diff
@@ -1,6 +1,6 @@
 {
-  "version": "1.0.0-*",
-  "description": "Razor precompilation",
+  "version": "1.1.0-*",
+  "description": "Razor view compilation",
```

Comment:
Something like this? `Razor view compilation for compiling CSHTML files into DLLs.`

Is that reasonably accurate?

Would we want to add even more, like this? `This package can help improve application startup time, detect Razor syntax errors at compilation time, and avoid the need to deploy CSHTML files to the server.`